tests/integration/test_typing.py

```diff
@@ -0,0 +1,221 @@
+import os
+import shlex
+import subprocess
+import sys
+
+import pytest
+
+component_template = """
+from dash_generator_test_component_typescript.TypeScriptComponent import TypeScriptComponent
+
+t = TypeScriptComponent({0})
+"""
+
+
```

Comment:
I don't think all of this should be necessary if we make pyright as part of the CI. Similar to flake8 or black.

Reply:
This tests the types are right, calling on the generated component would just tells you if the component has typing errors in it.